tests/test_data/fieldset_CROCO2D.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+import parcels
+
+
+def create_fieldset(indices=None):
```

Comment:
```suggestion
def create_fieldset():
```

Reply:
Oops, forgot to propagate indices as keyword argument to `parcels.FieldSet.from_croco()`